docker-run-opencv-ovms.sh

```diff
@@ -155,6 +155,14 @@ fi
 # Set GRPC port based on number of servers and clients
 GRPC_PORT=900$cid_count
 
+# Modify the config file if the device env is set
+# devices supported CPU, GPU, GPU.x, AUTO, MULTI:GPU,CPU
+if [ ! -z "$DEVICE" ]; then
+	echo "Updating config with device environment variable"
+	docker build -f `pwd`/configs/opencv-ovms/models/2022/Dockerfile.updateDevice -t update_config:dev `pwd`/configs/opencv-ovms/models/2022/.
```

Comment:
one issue: this should be built in the design time not the run time... it slows down if trying to build every time when you run

Reply:
I'll move this to the build-ovms-server make command since it is tied to that image.